{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,4 @@
-## For the Submitter
```

Comment:
I agree with @dan-f's point that it is better for the template to be more concise, so moving most of this material out seems like a good idea. I'm wondering if we could still have a very small section for the submitter. Maybe just a link to the CONTRIBUTING.md file and the explanation about what TODO means.

Another option would be to say that this information should be in the webhook versions, because then we could have a different explanation for internal developers and for OSPR tickets.

On the subject of OSPR, is there now duplication between the two templates that will appear on the page?